packages/near-fast-auth-signer/src/utils/multi-chain/signature/signature.ts

```diff
@@ -74,6 +64,46 @@ export const sign = async (
 
   const account = new Account(connection, nearAuthentication.accountId);
 
+  if (!relayerUrl) {
+    const multichainContractAcc = new Contract(account, contract, {
+      viewMethods:   [],
+      changeMethods: ['sign'],
+    }) as Contract & {
+      public_key: () => Promise<string>;
+      // eslint-disable-next-line no-unused-vars
+      sign: (args: {
+        args: {
+          payload: number[];
+          path: string;
+        };
+        gas: BN;
+      }) => Promise<[string, string]>;
+    };
+
+    const [R, s] = await multichainContractAcc.sign({
+      args: {
+        payload: Array.from(ethers.getBytes(transactionHash)).slice().reverse(),
+        path
+      },
+      gas: NEAR_MAX_GAS
+    });
+
+    return {
+      r: R.slice(2),
+      s
+    };
+  }
+
+  const functionCall = actionCreators.functionCall(
+    'sign',
+    {
+      payload: Array.from(ethers.getBytes(transactionHash)).slice().reverse(),
```

Comment:
`Array.from(ethers.getBytes(transactionHash)).slice().reverse()` used twice, lets pull it out to a constant?